packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -410,6 +412,28 @@ export async function* OpenAIChatModel(
     stream: true,
   };
 
+  function sleep() {
+    // return new Promise(resolve => setTimeout(resolve, 500));
+
+    // return new Promise<void>((resolve) => {
+    //   console.log('timeout start');
+    //   setTimeout(() => {
+    //     console.log('timeout done');
+    //     resolve();
+    //   }, 1000);
+    // })
```

Comment:
@petersalas when I did this, the whole app hung. This surprised me. Maybe we can sync and work through together why that may be happening.

Reply:
nm I figured it out